website/projects/scimaps/src/app/news-item/news-item.component.html

```diff
@@ -0,0 +1,8 @@
+<a class="news-item" [href]="item.pdfLink">
+  <img class="thumb" [src]="item.thumbnail" alt="image thumbnail">
+  <div class="info">
```

Comment:
All text without a font-size set automatically receives a font-size of 14px (and line height of 24px)